Sources/SwiftSyntax/Trivia.swift.gyb

```diff
@@ -215,6 +215,11 @@ public func +(lhs: Trivia, rhs: Trivia) -> Trivia {
   return Trivia(pieces: lhs.pieces + rhs.pieces)
 }
 
+/// Concatenates two collections of `Trivia` into the left-hand side.
+public func +=(lhs: inout Trivia, rhs: Trivia) {
+  lhs = lhs + rhs
+}
+
```

Comment:
I have a vague memory that from a type checking performance perspective, it’s better to implement these as static methods inside the `Trivia` type itself.

```suggestion
extension Trivia {
  /// Concatenates two collections of `Trivia` into the left-hand side.
  public static func +=(lhs: inout Trivia, rhs: Trivia) {
    lhs = lhs + rhs
  }
}

```

Reply:
Does this apply to the `+` operator too?

Reply:
It used to, but we removed the designated types optimization that supported it. Now it _should_ mostly a hygiene thing, and from that perspective I agree with Alex that this should be in extension context.